website/docs/intro.mdx

```diff
@@ -40,7 +40,7 @@ OSS Weave GitOps has a number of awesome features to take your team beyond a sim
 - :mag: Uncover relationships between resources and quickly navigate between them.
 - :thinking_face: Understand how workloads are reconciled through a directional graph.
 - :goggles: View Kubernetes events relating to a given object to understand issues and changes.
-- :no_pedestrians: Secure access to the dashboard through the ability to integrate with an OIDC provider (such as Dex) or through a configurable cluster user.
+- :no_pedestrians: Secure access to the dashboard through the ability to integrate with an OIDC provider (such as Dex).
```

Comment:
should we mention the emergency user mentioned above?

Reply:
I'd rather that nobody used it ever, so it's not a good feature to highlight in the intro imo